get_aws_secrets.sh

```diff
@@ -27,7 +27,7 @@ case "$secret_location" in
    ;;
 esac
 
-KMS_KEY_ID=`aws-okta exec $aws_account_profile -- aws kms list-aliases | grep -h1 $ACCOUNT_ALIAS\" | grep TargetKeyId | awk -F '\"' '{print $4}'`
+KMS_KEY_ID=`aws-okta exec $aws_account_profile -- aws kms list-aliases --output json| grep -h1 $ACCOUNT_ALIAS\" | grep TargetKeyId | awk -F '\"' '{print $4}'`
```

Comment:
Non blocking, if we are using json as output still worth to use grep instead JQ to or other kind filters to get the KMS_KEY_ID

Reply:
Nice idea, jq will simplify it. I will look into it.

Reply:
👍  At the end there I don't feel any value adding ` --output json` if we are going to manage it as regular text with grep

Reply:
I added it because in case a user has configured the AWS cli to have output set to yaml, the script runs ok but the KMS_KEY_ID is not populated, so the users can't create secrets. This way it doesn't matter if the output is set to yaml, text or klingon, we know it will work as we want. But yes, it makes total sense to swith to jq.

Reply:
I get your point know👏🏽